services/tenant-ui/frontend/src/views/connections/MyInvitations.vue

```diff
@@ -0,0 +1,7 @@
+<template>
+  <Invitations />
+</template>
+
+<script setup lang="ts">
+import Invitations from '../../components/invitations/Invitations.vue';
```

Comment:
May be good to start using "@" instead of relative paths.

Reply:
👍